plugins/generic/pln/templates/ping.tpl

```diff
@@ -0,0 +1,20 @@
+
+<plnplugin>
+    <ojsInfo>
+        <release>{$ojsVersion|escape}</release>
+    </ojsInfo>
+    <pluginInfo>
+        <release>{$pluginVersion.release}</release>
+        <releaseDate>{$pluginVersion.date}</releaseDate>
+        <installed>{$pluginVersion.version->getDateInstalled()}</installed>
+        <current>{$pluginVersion.version->getCurrent()}</current>
+    </pluginInfo>
+    <journalInfo>
+        <title>{$journal->getLocalizedTitle()}</title>
```

Comment:
Also beware of getLocalized... functions -- they choose a locale based on the user's current locale, which can lead to different XML extracts depending on the environment they're fetched from. See a similar comment with some details here: https://github.com/pkp/ojs/pull/388#discussion_r22967375